pkg/syncer/metadatasyncer.go

```diff
@@ -2958,6 +2958,12 @@ func storagePolicyUsageCRSync(ctx context.Context, metadataSyncer *metadataSyncI
 		log.Errorf("Failed to get PVs from kubernetes. Err: %+v", err)
 		return
 	}
+	// Get K8s PVCs not in Bound State.
+	k8sPVCsInPendingState, err := getPVCsInPendingState(ctx, metadataSyncer)
```

Comment:
Can you update the comments at line 2951 accordingly for bound PVCs?

Reply:
The method doc does have comments for both Bound and not Bound PVCs. Anything specific you wanted me to add there ? Let me know

Reply:
The comments are as follows:
```
// storagePolicyUsageCRSync patches StoragePolicyUsage CRs for k8s PVs in Bound state.
// This method also creates StoragePolicyUsage CRs when the CR is not found for PVs in Bound state.
```
It does not mention unbound PVCs?

Reply:
Right, my bad. I missed the unbound/pending PVCs. Updated the doc